src/MacroTools/Commands/Share.cs

```diff
@@ -0,0 +1,45 @@
+﻿using MacroTools.CommandSystem;
+using MacroTools.Extensions;
+using MacroTools.FactionSystem;
+using static War3Api.Common;
+
+namespace MacroTools.Commands
+{
+  /// <summary>Share control of your units with another player.</summary>
+  public sealed class Share : Command
+  {
+    /// <inheritdoc />
+    public override string CommandText => "share";
```

Comment:
Might be being pedantic here but would capitalise this so it displays "Share: Shared control with x" rather than "share: Shared control with x"

Reply:
You won't see me complaining about useful pedantry. However I tried this change out and I think it's better to use the lowercase version, since it's closer to what the command actually is.